src/main/java/seedu/address/model/person/exceptions/DuplicateEmployeeException.java

```diff
@@ -3,10 +3,10 @@
 import seedu.address.commons.exceptions.DuplicateDataException;
 
 /**
- * Signals that the operation will result in duplicate Person objects.
+ * Signals that the operation will result in duplicate Employee objects.
  */
-public class DuplicatePersonException extends DuplicateDataException {
-    public DuplicatePersonException() {
+public class DuplicateEmployeeException extends DuplicateDataException {
+    public DuplicateEmployeeException() {
         super("Operation would result in duplicate persons");
```

Comment:
Doesn't seems very useful to me unless you're showing the message to user, asking for confirmation.